### PR TITLE
Basic server setup including DB setup and gem files

### PR DESCRIPTION
**Pull request by Garo-A**

### CODE REVIEW

**Comment by Garo-A** [comment]
Don't forget to `gem install` when you guys pull! Added a few necessary packages to Gemfile

File: Gemfile [hunk 1]
```diff
@@ -30,14 +30,22 @@ gem 'jbuilder', '~> 2.5'
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
+gem 'bcrypt', '~> 3.1.7'
+
+gem 'bootstrap-sass', '~> 3.3.6'
+gem 'font-awesome-rails'
+gem 'money-rails'
```
Comment (DerangedMind) [inline Gemfile]: Do we need money-rails?

File: Gemfile [hunk 2]
```diff
@@ -30,14 +30,22 @@ gem 'jbuilder', '~> 2.5'
...
+gem 'carrierwave'
```
Comment (DerangedMind) [inline Gemfile]: I'm not seeing a purpose for this right now. Let's wait on it?

File: config/database.yml [hunk 3]
```diff
@@ -5,21 +5,22 @@
 #   gem 'sqlite3'
```
Comment (DerangedMind) [inline config/database.yml]: We're not using sqlite3 anymore! Change to gem 'pg'

**Comment by DerangedMind** [comment]
👍

**Pull request merged**